org.nongnu.gsequencer.gsequencer.json

```diff
@@ -212,15 +254,32 @@
                 }
             },
             "post-install": [
+                "mkdir -p /app/share/doc/gsequencer/pdf/",
+                "install -c -p -m 644 pdf/user-manual-a4.pdf /app/share/doc/gsequencer/pdf/user-manual-a4.pdf",
+                "install -c -p -m 644 pdf/user-manual-letter.pdf /app/share/doc/gsequencer/pdf/user-manual-letter.pdf",
                 "sed -i -e 's~Icon=gsequencer~Icon=org.nongnu.gsequencer.gsequencer~g' /app/share/applications/gsequencer.desktop",
```

Comment:
there is `desktop-file-edit` to do that kind of changes, less obscure than `sed`

also with `rename-icon` it's not even necessary.

Reply:
Removed it.